com.unity.netcode.gameobjects/Samples~/Bootstrap/Scripts/BootstrapManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Unity.Netcode;
+using UnityEngine;
+
+namespace BoostrapSample
+{
+    /// <summary>
+    /// Class to display helper buttons and status labels on the GUI, as well as buttons to start host/client/server.
+    /// Once a connection has been established to the server, the local player can be moved via a GUI button.
+    /// </summary>
+    public class BootstrapManager : MonoBehaviour
+    {
+        void OnGUI()
```

Comment:
I think we should apply coding standards to these source files as well.
I wonder why it passed CI checks, maybe because it doesn't create a csproj or sln file?

Reply:
we had a conversation with Fernando. there are couple of extra steps to make samples compile and also eventually part of CI standards check. I'll create another JIRA ticket for this issue and address those over there.